README.md

```diff
@@ -66,6 +66,7 @@ const saml = new SAML(options);
 - `disableRequestAcsUrl`: if truthy, SAML AuthnRequest from the service provider will not include the optional AssertionConsumerServiceURL. Default is falsy so it is automatically included.
 - `generateUniqueId`: optional function which will be called to generate unique IDs for SAML requests.
 - `scoping`: An optional configuration which implements the functionality [explained in the SAML spec paragraph "3.4.1.2 Element <Scoping>"](https://docs.oasis-open.org/security/saml/v2.0/saml-core-2.0-os.pdf). The config object is structured as following:
+- `signMetadata`: if truthy, adds a signature to the generated Service Provider metadata. `privateKey` must be set to use this option.
```

Comment:
I think I'd like to tighten up things as we go forward on this project to use `true` instead of truthy.

Reply:
👍